css-two-column-layout/styles.css

```diff
@@ -0,0 +1,95 @@
+* {
+  font-family: karla, monospace;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1000px;
+}
+
+.main-header,
+.sub-header,
+button {
+  text-transform: uppercase;
+}
+
+.main-header {
+  flex-basis: 100%;
+  text-align: center;
+  margin-top: 20px;
+  font-size: 150%;
+  font-weight: bold;
+  letter-spacing: 0.1rem;
+}
+
+.main-header-description {
+  flex-basis: 70%;
+  text-align: center;
+  margin-top: -30px;
+  margin-bottom: 40px;
+  letter-spacing: -2%;
+  line-height: 120%;
+}
+
+.main-header-description,
+.caption {
+  line-height: 150%;
+  color: rgb(105, 105, 105);
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: 100%;
+  justify-content: center;
+}
+
+.quadrant {
+  flex-basis: 50%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0;
+}
+
+.image {
+  flex-basis: 20%;
+  width: 20%;
+}
+
+img {
+  width: 95%;
+  height: 95%;
+  position: relative;
+  bottom: 33%;
+}
+
+.text-content {
+  display: flex;
+  flex-wrap: wrap;
+  width: 60%;
+}
+
+.sub-header {
+  width: 15rem;
+}
+
+.caption {
+  margin-top: -20px;
```

Comment:
In general negative margins should be avoided. If you feel that you need a negative margin, that usually just means that an adjacent element's margin should be reduced.